06-reactjs/components/input.js

```diff
@@ -0,0 +1,40 @@
+var React = require('react');
+var _ = require('lodash');
+
+
```

Comment:
**So-minor-that-I'm-ashamed-to-mention-it:** remove extra line break on line 4. :P